63_llm_alpha_mining/python/alpha_generator.py

```
@@ -0,0 +1,694 @@
+                }
+            ]
+
+        return json.dumps(factors, indent=2)
+
+    def generate(
+        self,
+        data: pd.DataFrame,
+        prompt_type: str = "basic",
+        symbol: str = "UNKNOWN",
+        market_type: str = "crypto",
+        n_factors: int = 3,
```

Ruff (0.14.13)

Unused method argument: `n_factors`

(ARG002)